apps/client/src/pages/home/components/performance-carousel-section.tsx

```diff
@@ -1,42 +1,62 @@
-
-import { useNavigateToDetail } from '@shared/hooks/use-navigate-to-detail';
-import { CarouselPerformancesResponse } from '@shared/types/home-response';
-import { convertToCdnUrl } from '@shared/utils/convert-to-cdn-url';
+import { useCarouselBackground } from '../hooks/use-carousel-background';
+import PerformanceCarousel from './performance-carousel/components/performance-carousel';
+import { CAROUSEL_ANIMATION } from './performance-carousel/constants/animation';
+import { type Performance } from './performance-carousel/types/performance-carousel-types';
 
 import * as styles from './performance-carousel-section.css';
 
+interface PerformanceCarouselSectionProps {
+  data: Performance[];
+  isPersonalized: boolean;
+  onPerformanceClick?: (type: 'FESTIVAL' | 'CONCERT', typeId: number) => void;
+}
+
 const PerformanceCarouselSection = ({
   data,
-}: {
-  data: CarouselPerformancesResponse;
-}) => {
-  const navigateToDetail = useNavigateToDetail();
-
-  const badgeText = data.isPersonalized ? '선호하는 아티스트' : '다가오는 공연';
-  const displayPerformances = data.performances.slice(0, 7);
-  const formattedPerformData = displayPerformances.map((performance) => {
-    return {
-      ...performance,
-      performanceAt: formatDate(performance.startAt),
-      posterUrl: convertToCdnUrl(performance.posterUrl),
-    };
-  });
-  const initialSlideIndex = Math.floor(formattedPerformData.length / 2);
+  isPersonalized,
+  onPerformanceClick,
+}: PerformanceCarouselSectionProps) => {
+  const { activeIndex, nextIndex, isTransitioning, handleSlideChange } =
+    useCarouselBackground();
+
+  if (!Array.isArray(data) || data.length === 0) return null;
 
   return (
-    <section className={styles.performanceBannerContainer}>
-      <PerformanceCarousel
-        performData={formattedPerformData}
-        initialSlideIndex={initialSlideIndex}
-        handleContainerClick={(type, typeId) => navigateToDetail(type, typeId)}
-      >
-        <PerformanceCarousel.ImageSlider>
-          <PerformanceCarousel.Badge text={badgeText} />
-          <PerformanceCarousel.Info />
-        </PerformanceCarousel.ImageSlider>
-      </PerformanceCarousel>
+    <section className={styles.sectionContainer}>
+      <div className={styles.backgroundWrapper}>
+        {/* 현재 배경 */}
+        <img
+          src={data[activeIndex]?.posterUrl}
+          alt={data[activeIndex]?.title}
+          className={styles.backgroundImage}
+          style={{
+            transition: `opacity ${CAROUSEL_ANIMATION.DURATION}ms ${CAROUSEL_ANIMATION.EASING}`,
+          }}
+        />
+
+        {/* 다음 배경 (fade-in) */}
+        <img
+          src={data[nextIndex]?.posterUrl}
+          alt={data[nextIndex]?.title}
+          className={styles.backgroundImageFront}
+          style={{
+            opacity: isTransitioning ? 1 : 0,
+            transition: `opacity ${CAROUSEL_ANIMATION.DURATION}ms ${CAROUSEL_ANIMATION.EASING}`,
+          }}
+        />
```

Comment:
depth가 조금 생긴다고 느낄 수 있지만 주석 없이 의미를 전달하게
```suggestion
        {/* 현재 배경 */}
        <CurrentImage  .../>

        {/* 다음 배경 (fade-in) */}
        <NextImage .../>
```
이렇게 돼도 좋을 것 같아요